java/src/org/openqa/selenium/manager/SeleniumManager.java

```diff
@@ -147,20 +148,15 @@ private File getBinary() {
 
   /**
    * Determines the location of the correct driver.
-   * @param driverName which driver the service needs.
+   * @param options Browser Options instance.
    * @return the location of the driver.
    */
-    public String getDriverPath(String driverName) {
-        if (!ImmutableList.of("geckodriver", "chromedriver", "msedgedriver", "IEDriverServer").contains(driverName)) {
-            throw new WebDriverException("Unable to locate driver with name: " + driverName);
-        }
-
-        String driverPath = null;
+    public String getDriverPath(Capabilities options) {
```

Comment:
The public method's signature is being changed. Should there be a deprecate instead and a new method? I might be missing the context/usage here.

Reply:
Nevermind, figured out the internal usage.